src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1459,4 +1461,50 @@ private function addMailerSection(ArrayNodeDefinition $rootNode)
             ->end()
         ;
     }
+
+    private function addNotifierSection(ArrayNodeDefinition $rootNode)
+    {
+        $rootNode
+            ->children()
+                ->arrayNode('notifier')
+                    ->info('Notifier configuration')
+                    ->{!class_exists(FullStack::class) && class_exists(Notifier::class) ? 'canBeDisabled' : 'canBeEnabled'}()
```

Comment:
Should we get rid of the FullStack class in 5.0?

Reply:
I tried, I opened a PR, but it gets messy pretty fast, I gave up, not worth it.